### DISCUSSION TITLE
Error opening configuration file

### DISCUSSION THREAD

**Discussion by MajorLOL** [qdm12/gluetun#662] [Q&A]
Greetings,

Currently receiving the following message:

2021/10/05 17:58:43 ERROR openvpn: In [CMD-LINE]:1: Error opening configuration file: /etc/openvpn/target.ovpn
2021/10/05 17:58:43 INFO openvpn: Use --help for more information.

Anyone that know how to fix this? Currently it is not running.
I am on Synology 7.0 running docker 20.10.3-1233
![image](https://user-images.githubusercontent.com/56225967/136078516-9fb50e86-1c76-48f9-96bf-7d6d22d3d156.png)
![image](https://user-images.githubusercontent.com/56225967/136078627-2b5583d3-f02f-47e3-964c-fac5beedf844.png)
![image](https://user-images.githubusercontent.com/56225967/136078738-33af71d9-ee6b-4f15-9ec7-f4afe3d1ef17.png)

**Comment by qdm12** [comment]
1. Can you find the Gluetun version you are running (from the top of the logs) please?
2. Do you have any bind mounts? It looks like it doesn't manage to open the (built at start/runtime) OpenVPN configuration `/etc/openvpn/target.ovpn`, I'm not sure why.
3. When it works again, you should set `FIREWALL=on` 😉

**Comment by MajorLOL** [comment]
Greetings, Thank you for helping, 
Here is my version: 
Running version latest built on 2021-10-05T17:48:06Z (commit e0e3ca3)

my mounts are:
/volume1/docker/gluetun | /gluetun
/volume1/docker/gluetun/target.ovpn | /etc/openvpn/target.ovpn

Tryed it without:
/volume1/docker/gluetun/target.ovpn | /etc/openvpn/target.ovpn

But did same results, I am kind of a loss on what to do at this point.

**Reply by qdm12** [reply]
Don't bind mount `/etc/openvpn/target.ovpn` or `/etc/openvpn`. Did you do that by yourself or is the default in the Unraid template?

Maybe try removing the container and then start it (not sure how to do it on Unraid though).

**Reply by MajorLOL** [reply]
That seems to fix the problem.
